packages/core/src/theme.js

```diff
@@ -42,7 +42,7 @@ export const fontWeights = {
 }
 
 export const lineHeights = {
-  standard: 1.5,
+  standard: 1.4,
```

Comment:
So this was never supposed to be 1.5?

Reply:
Would it be a breaking change to change the line-height now?  Should this wait until 3.0.0?

Reply:
@jes708, shared more offline to explain about the expect 1.5, but it's the intended update and no style effect as this standard pellet of line height is not really used by anything.